umpost/um2netcdf.py

```diff
@@ -92,46 +105,194 @@ def convert_proleptic(time):
     time.units = newunits
 
 
-def fix_latlon_coord(cube, grid_type, dlat, dlon):
-    def _add_coord_bounds(coord):
-        if len(coord.points) > 1:
-            if not coord.has_bounds():
-                coord.guess_bounds()
-        else:
-            # For length 1, assume it's global. guess_bounds doesn't work in this case
-            if coord.name() == 'latitude':
-                if not coord.has_bounds():
-                    coord.bounds = np.array([[-90., 90.]])
-            elif coord.name() == 'longitude':
-                if not coord.has_bounds():
-                    coord.bounds = np.array([[0., 360.]])
+def fix_lat_coord_name(lat_coordinate, grid_type, dlat):
+    """
+    Add a 'var_name' attribute to a latitude coordinate object
+    based on the grid it lies on.
+
+    Parameters
+    ----------
+    lat_coordinate: coordinate object from iris cube (edits in place).
+    grid_type: (string) model horizontal grid type.
+    dlat: (float) meridional spacing between latitude grid points.
+    NB - Only applies to variables on the main horizontal grids, 
```

Comment:
Small thing, I noticed the "NB" in these param docs.

Reply:
Good catch, should be fixed now!